get-compose-running.sh

```diff
@@ -26,7 +26,7 @@ containers_running() {
         if [ ${is_dead} -eq 1 ]; then
             consumer_name=$(docker ps -a --filter="id=${id}" --format="{{ .Names }}")
             echo -e "\nERROR: Container '$consumer_name' is DEAD\n"
-            docker logs ${id}
+            docker logs ${id} | tail -n100
```

Comment:
This could be done using the `--tail 100` docker logs flag, but this should do the trick. ;)